test/run_test.py

```diff
@@ -14,6 +14,7 @@
 import sys
 import tempfile
 import json
+import multiprocessing as mp
```

Comment:
Do you need to use the stock multiprocessing module here?  There is already `from torch.multiprocessing import Pool` imported.  The latter is built on top of multiprocessing